server/models/Integration.ts

```diff
@@ -27,15 +55,18 @@ import Fix from "./decorators/Fix";
 @Table({ tableName: "integrations", modelName: "integration" })
 @Fix
 class Integration extends IdModel {
+  @IsIn([Object.values(IntegrationType)])
   @Column
   type: string;
 
+  @IsIn([Object.values(IntegrationService)])
   @Column
   service: string;
 
   @Column(DataType.JSONB)
-  settings: Record<string, any>;
+  settings: IntegrationSettings;
 
+  @IsIn([["documents.update", "documents.publish"]])
```

Comment:
Why was this added @tommoor ?

Reply:
I was just adding validation for existing expectations, is it a requirement with this in place or still nullable?

Reply:
What if no events are sent? Or are `events` always required?

Reply:
I'm not sure without checking but they don't _need_ to be required.

Reply:
I'm not sure without checking but they don't _need_ to be required.